samples/ExtendedUWPSamples/MSDKInterop/StreamWriter.cpp

```diff
@@ -0,0 +1,43 @@
+/******************************************************************************\
```

Comment:
That's a duplicate from sample_common. Why do we need a duplicate? If that was trivial 10 lines of code that would be ok, but it is not.... That's hundreds of lines, multiple files. From now on we will need to support same code in 2 places. Were there issues you've fixed? Refactoring done? Why you can't update, improve and reuse sample_common?

Reply:
These samples were initially designed as standalone samples package so we could provide the code with minimal dependencies.